src/main/java/org/commonwl/view/workflow/WorkflowController.java

```diff
@@ -549,4 +582,11 @@ private ModelAndView getWorkflow(GitDetails gitDetails, RedirectAttributes redir
                     WebConfig.Format.values());
         }
     }
+
+    private Resource getGraphFromInputStream(InputStream in, String format)
+            throws IOException, NoSuchAlgorithmException {
+        Workflow workflow = cwlService.parseWorkflowNative(in, "", "workflow"); // first workflow will do
```

Comment:
@stain I think here `packedWorkflowId` needs to be null. Otherwise posting via `curl` results in Workflow not found, because the `parseWorkflowNative` will check `if (packedWorkflowId != null)` and will throw `WorkflowNotFoundException` since it couldn't locate the packed workflow.

```suggestion
        Workflow workflow = cwlService.parseWorkflowNative(in, null, "workflow"); // first workflow will do
```